evolver/calibration/interface.py

```diff
@@ -138,6 +151,7 @@ def model_post_init(self, __context: Any) -> None:
     def __init__(self, *args, calibration_file=None, **kwargs):
         super().__init__(*args, calibration_file=calibration_file, **kwargs)
         self.calibration_procedure = None
+        self.calibration_data = self.CalibrationData()
```

Comment:
This assumes all fields of ``CalibrationData`` are optional. Is this a correct restriction?

Reply:
yeah for the time being all the fields on `Transformer` and are optional.